### DISCUSSION TITLE
Verständnisfrage Pin-Belegung

### DISCUSSION THREAD

**Discussion by Troubadix** [tbnobody/OpenDTU#2274] [General]
Hallo,

ich werde leider aus der Doku nicht schlau und bräuchte mal einen Denkanstoß...

https://www.opendtu.solar/firmware/device_profiles/

Woher kommen die Zahlen?
z.B. für ein Display steht in der pin_mapping.json "data = 33". Da kommt ja dann der SDA-Pin vom Display dran, richtig?

Woher weiß ich, was Pin 33 ist? Der Olimex POE ESP32 selbst hat ja nur 20 Pins, muss also eine andere Zählweise sein?

Danke...

**Comment by ms49434** [comment]
Es ist nicht Pin 33 gemeint, sondern der GPIO mit der Nummer 33 des ESP32 Chips. Es hängt vom Designer des Platinenlayouts ab, welcher GPIO auf welchem Pin zur Verfügung steht.

**Comment by Troubadix** [comment]
Aaaah GPIO! Schreib das doch mal einer dazu, überall liest mal nur von Pinbelegung :D
Danke!

**Reply by Solarteur** [reply]
In der Doku (oben zitiert) steht:

_"Please be aware that numerical values used in profile .json file are the ESP chip GPIO numbers (NOT phy. PIN numbers)."_

**Reply by Troubadix** [reply]
Oh man, da war ich schön blind... ich musste jetzt auch wirklich über "Auf der Seite suchen" nach dem Text suchen, da ich das nicht gesehen habe.... Ich Depp. Danke.
